server/block/cactus.go

```diff
@@ -58,13 +58,17 @@ func (c Cactus) NeighbourUpdateTick(pos, _ cube.Pos, w *world.World) {
 
 // RandomTick ...
 func (c Cactus) RandomTick(pos cube.Pos, w *world.World, r *rand.Rand) {
-	if c.Age < 15 && r.Float64() < 0.01 {
-		abovePos := pos.Add(cube.Pos{0, 1})
+	if c.Age < 15 {
+		c.Age++
```

Comment:
@DaPigGuy would you know why this line is marked as "ineffective assignment to field Cactus.Age (SA4005)"?